kolibri/plugins/coach/assets/src/views/exams/exam-report-page/index.vue

```diff
@@ -33,14 +33,10 @@
         :key="i"
       >
         <h3>
-          {{ viewByGroups ? reportGrouping[0].group.name : $tr('allLearners') }}
+          {{ viewByGroups ? reportGrouping[0].group.name || $tr('ungrouped') : $tr('allLearners') }}
```

Comment:
Appears to cause a problem if `reportGrouping[0]` is undefined, which can happen if you have a Learner Group with zero members.